webcompat/static/js/lib/wizard/utils.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const HEADER_HEIGHT = 130;
+
+const utils = {
+  scrollToElement: (element, delay = 250) => {
+    // Delay "scroll to element" effect in order to let the animation finish, otherwise the scroll point isn't correct
+    setTimeout(() => {
+      const topOfElement =
+        window.pageYOffset +
+        element.getBoundingClientRect().top -
+        HEADER_HEIGHT;
+      window.scroll({ top: topOfElement, behavior: "smooth" });
+    }, delay);
+
+    //@todo figure out why this is needed
+    //this.isSubproblem ? 450 : 250
```

Comment:
(since it's commented out, is it really needed 🤔)

Reply:
I've moved this from existing script and decided to leave it since not quite sure why it was used, planning on revisiting this..